90captive_portal_test

```diff
@@ -1,49 +1,44 @@
 #!/bin/sh -e
```

Comment:
If you're going to use `local` you have to change to `#!/bin/bash -e`

Reply:
dash and busybox ash do have it